src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java

```diff
@@ -0,0 +1,178 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.google.common.collect.Lists;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
```

Comment:
I value consistency here, and we've had no troubles with Joda that would lead us to switch over. In fact they are so similar that using both could lead to a lot of confusion.